### ISSUE TITLE
context in decorator is undefined

### ISSUE THREAD

**Issue by ColCh** [kadirahq/react-native-storybook#82]
Trying to run knobs with react-native-storybook. Got an error:

```
2016-09-25 14:36:39.159 [fatal][tid:com.facebook.react.RCTExceptionsManagerQueue] Unhandled JS Exception: Cannot read property 'kind' of undefined
```

![image](https://cloud.githubusercontent.com/assets/599439/18815070/d46299a0-832d-11e6-84be-bd859f8f6cad.png)

A bit debugging:
![image](https://cloud.githubusercontent.com/assets/599439/18815074/f1a3ca16-832d-11e6-9bf9-7ea124e608a5.png)

**Comment by thani-sh** [comment]
Hi @ColCh 
Thanks for reporting this bug, I've opened a pull-request with a fix but unfortunately I'm unable to test it today. I'll merge this and do a release as soon as I could verify.

@arunoda @roonyh 
If you guys can check this out, let me know or just merge the PR and do a release.

**Comment by ColCh** [comment]
@mnmtanish thanks. applied this fix in local `node_modules`, it works nice now

But then... issue with `knobs` itself: https://github.com/kadirahq/storybook-addon-knobs/issues/41

**Comment by thani-sh** [comment]
Released `v2.1.2`
